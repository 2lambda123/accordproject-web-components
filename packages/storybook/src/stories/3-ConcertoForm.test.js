import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /Concerto Form/,
  suite: 'Concerto Form Storyshots'
});
