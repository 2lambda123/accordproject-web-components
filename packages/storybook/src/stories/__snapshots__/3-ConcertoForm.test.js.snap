// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Concerto Form Storyshots Concerto Form Demo 1`] = `
<div
  style={
    Object {
      "padding": "10px",
    }
  }
>
  <div
    className="ui active transition visible inverted dimmer"
    onClick={[Function]}
  >
    <div
      className="content"
    >
      <div
        className="ui inverted text loader"
      >
        Loading
      </div>
    </div>
  </div>
</div>
`;
